docker-compose.yml

```diff
@@ -1,21 +1,37 @@
 ---
 version: '3'
 services:
+  minica:
+    image: registry.access.redhat.com/ubi8/go-toolset:latest
```

Comment:
Could you add the comments why is mimica needed?

Reply:
Good idea. Done.